tests/deprecation/test_middleware_mixin.py

```diff
@@ -54,6 +54,29 @@ class MiddlewareMixinTests(SimpleTestCase):
         XViewMiddleware,
     ]
 
+    def test_repr(self):
+        class GetResponse:
+            def __call__(self):
+                return HttpResponse()
+
+        def get_response():
+            return HttpResponse()
+
+        for middleware in self.middlewares:
```

Comment:
We added `__repr__()` to the `MiddlewareMixin` so it should be enough to test only one of its subclasses.